README.md

```diff
@@ -1950,14 +1950,13 @@ to taxes on equity compensation:
 
 ### Key aspects of your equity
 
-If you’re familiar with the types of equity and how it relates to taxes we’ve covered in
-previous sections, you now have many of the tools you need to evaluate an offer that
-includes equity compensation, or to evaluate equity you currently have in a company.
-
-- In determining the value of equity, you have to figure out (or, more accurately, make
-  educated guesses about) several things.
-  We can start measuring the potential value of equity by doing our best to determine the
-  following:
+Now that you’re familiar with the [types of equity](#how-equity-is-granted) and how they
+[relate to taxes](#taxes-on-equity-compensation), you have many of the tools you need to
+evaluate an offer that includes equity compensation, or to evaluate equity you currently
+have in a company.
+
+- To measure the potential value of equity, you have to determine (or, more accurately, make
```

Comment:
Tightening: "you have to determine or make educated guesses about several things"

Reply:
Fixed.